bot/exts/fun/quack.py

```diff
@@ -0,0 +1,75 @@
+import random
+from typing import Optional
+
+import discord
+from aiohttp import client_exceptions
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+API_URL = 'https://quackstack.pythondiscord.com'
+
+
+class Quackstack(commands.Cog):
+    """Cog used for wrapping Quackstack."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command()
+    async def quack(self, ctx: commands.Context, ducktype: str = "duck", *, seed: Optional[str] = None) -> None:
```

Comment:
```suggestion
    async def quack(self, ctx: commands.Context, ducktype: Literal["duck", "manduck"] = "duck", *, seed: Optional[str] = None) -> None:
```
According to @Numerlor this should work as a converter. (don't forget to import `Literal` from `typing`)